cmd/gatekeeper-test-alpha/gatekeeper-test-alpha.go

```diff
@@ -1,16 +1,54 @@
 package main
 
 import (
+	"errors"
 	"fmt"
 	"os"
 
+	"github.com/open-policy-agent/gatekeeper/pkg/gktest"
 	"github.com/spf13/cobra"
 )
 
+var run string
+
+func init() {
+	rootCmd.Flags().StringVarP(&run, "run", "r", "",
+		`regular expression which filters tests to run by name`)
+}
+
 var rootCmd = &cobra.Command{
 	Use:   "gatekeeper-test-alpha path [--run=name]",
 	Short: "Gatekeeper Test Alpha is a unit test CLI for Gatekeeper Constraints",
+	Example: `  gatekeeper-test-alpha label-tests.yaml
+  gatekeeper-test-alpha tests/... --run forbid-labels`,
+	Version: "alpha",
+	Args:    cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		path := args[0]
+
+		testFiles, err := gktest.ToTestFiles(path)
+		if err != nil {
+			return err
+		}
+		filter, err := gktest.NewFilter(run)
+		if err != nil {
+			return err
+		}
+
+		isFailure := false
+		for _, f := range testFiles {
+			result := gktest.Run(f, filter)
+			// If Result contains an error status, it is safe to execute tests in other
+			// files so we can continue execution.
+			isFailure = isFailure || result.IsFailure()
+			fmt.Println(result.String())
+		}
+		if isFailure {
+			// At least one test failed or there was a problem executing tests in at
+			// least one file.
+			return errors.New("FAIL")
```

Comment:
This will probably be added later. It would be good to print out the failure for each failed test as soon as they are done running.

Reply:
Yep. I don't know how long individual test suites will take, so I may want to allow parallel execution. In this case, I'm following the Go pattern of printing test output as they finish, and then ending with a "FAIL" line at the very end.